src/validate.js

```diff
@@ -152,13 +156,20 @@ export default function validateFormData(
   formData,
   schema,
   customValidate,
-  transformErrors
+  transformErrors,
+  additionalMetaSchemas
 ) {
+  // add more schemas to validate against
+  if (!addedMetaSchemas && additionalMetaSchemas) {
+    ajv.addMetaSchema(additionalMetaSchemas);
+    addedMetaSchemas = true;
+  }
+  let validationErrors = null;
   try {
     ajv.validate(schema, formData);
-  } catch (e) {
-    // swallow errors thrown in ajv due to invalid schemas, these
-    // still get displayed
+  } catch (err) {
+    console.error(err);
```

Comment:
only problem with this console.error is that in test in few places we have now 
![image](https://user-images.githubusercontent.com/17271979/52045085-01d12b80-2544-11e9-8854-9eb7506e1876.png)

One such error is in my test (intentional), two are in old tests